src/common/dao/group/usergroup.go

```diff
@@ -101,17 +101,17 @@ func GetUserGroup(id int) (*models.UserGroup, error) {
 // GetGroupIDByGroupName - Return the group ID by given group name. it is possible less group ID than the given group name if some group doesn't exist.
 func GetGroupIDByGroupName(groupName []string, groupType int) ([]int, error) {
 	var retGroupID []int
-	var conditions []string
+	var qm []string
 	if len(groupName) == 0 {
 		return retGroupID, nil
 	}
-	for _, gName := range groupName {
-		con := "'" + gName + "'"
-		conditions = append(conditions, con)
+	for i := 0; i < len(groupName); i++ {
+		qm = append(qm, "?")
 	}
```

Comment:
There is a function `ParamPlaceholderForIn` can do this.

Reply:
Updated